src/middleware/stats.ts

```diff
@@ -0,0 +1,29 @@
+import autoBind from "auto-bind"
+
+import {
+  statsService as statsServiceInstance,
+  StatsService,
+} from "@root/services/infra/StatsService"
+import { RequestHandler } from "@root/types"
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
oh i set it here initially to remove the squiggles. i'll remove it now but the reason why it's not a default export here is because there's a few exports. i don't feel too strongly about this so if you want, i can add the default export back in